Makefile

```diff
@@ -99,6 +99,15 @@ ci.aws-acceptance-test-cleanup: ## Deletes AWS resources left behind after faile
 version:
 	@echo $(VERSION)
 
+# ===========> Release Targets
+
+prepare-release: ## Sets the versions, updates changelog to prepare this repository to release
+ifndef RELEASE_VERSION
+	$(error RELEASE_VERSION is required)
+endif
+	source $(CURDIR)/control-plane/build-support/scripts/functions.sh; set_release_mode $(CURDIR) $(RELEASE_VERSION) "$(shell date +"%B %d, %Y")" $(PRERELEASE_VERSION)
```

Comment:
(Not blocking)  

I was just thinking that it might be useful to be able to set the date so that we could pre-load the PR ahead of time?  I guess, whomever is releasing could set the date in the PR if they are doing it early....

Reply:
Hmm yeah. I'd hold off on this until we need it. Typically we end up doing it the same day IIRC. Definitely easy to improve in the future!